### ISSUE TITLE
Display Results

### ISSUE THREAD

**Issue by stdavis** [agrc/api-client#13]
It would be nice if we displayed at least a selection of the output CSV for the user to review after the geocoding process is complete.

**Comment by stdavis** [comment]
Or perhaps we should show a selection during geocoding so that someone doesn't sit through six hours of geocoding only to discover that the output wasn't correct:

>Just did my first batch of about 60,000 records. It went really well, good matches, with a match rate of about 88%, I did see a lot of the nonmatches were homeless, which is to be expected, so my match rate is even higher. I have to run it again, when I convert the file to CSV the trailing zeros in the recipient ID is removed, so I can't link it back. Now I am just creating a unique identifier that won't have that issue and rerunning it. So, not bad for the first attempt. It takes about six hours to run, so I will be running that again today.

**Comment by steveoh** [comment]
Agreed but it still might be tricky to notice a missing 0 on an id. It seems that wasn't our fault though.
